server/monitor.go

```diff
@@ -1038,6 +1038,8 @@ func (reason ClosedState) String() string {
 		return "Route Removed"
 	case ServerShutdown:
 		return "Server Shutdown"
+	case LameDuckMode:
```

Comment:
Agree now just use server shutdown as reason. No need for new one.

Reply:
Ok, can revert this change